index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<head>
+    <title>ILC 5</title>
+    <link rel="stylesheet" type="text/css" href="js/css/style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+    <script src="js/script.js"></script>
```

Comment:
you are missing the jquery ui script and css - as well as the accounting js.
This will mean lots of stuff wont' work.
-2